Python/pywarpx/_libwarpx.py

```diff
@@ -625,6 +627,9 @@ def add_particles(self, species_name, x=None, y=None, z=None, ux=None, uy=None,
             # --- The -3 is because components 1 to 3 are velocities
             attr[:,self.get_particle_comp_index(species_name, key)-3] = vals
 
+        nattr_int = 0
+        attr_int = np.empty([0], ctypes.c_int)
```

Comment:
This is not necessarily needed for this PR but we should at some point improve the logic here to allow both real and integer attributes to be set via `add_particles`. This would require adding a function to `WarpXWrappers` to link to `getParticleiComps()`. This implementation somewhat moves the problem this PR is fixing on to Python specific injection i.e. integer runtime components will silently be set to 0 for newly injected particles. Maybe just creating a corresponding Issue is sufficient for now to remind us of this shortcoming.

Reply:
Yes, this does not fix the integer runtime attributes initialization from Python, but they were already broken, right? (I just want to make sure that I'm not breaking anything in your workflows). I'd agree to say that this can go to a follow-up PR.

Reply:
That is correct, yes, it was already broken. This PR just made it clear so documenting the issue now would be good so we don't have to learn about it the hard way again later on.

Reply:
Note that with the new implementation, the int attributes should be correctly set with python (but there's no way to initialize them through a numpy array yet).